### PR TITLE
personaldetailspopup

### PR DESCRIPTION
**Pull request by infanta-deriv**
card - https://app.clickup.com/t/20696747/TRAH-3050
This PR has script for validating the functionality when user closes the "complete your personal details" pop up during sign up

### CODE REVIEW

File: cypress/e2e/wip/Tradershub/personalDetailsPopUpValidation.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import "@testing-library/cypress/add-commands"
+import {generateEpoch} from '../../../support/tradersHub'
+
+  describe("QATEST-24444 - Verify the user is able to close the personal details pop up during sign up", () => {
+    const epoch = generateEpoch()
+    const sign_up_mail = `sanity${epoch}account@deriv.com`
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wip/Tradershub/personalDetailsPopUpValidation.js:6]: please convert to camelCase

File: cypress/e2e/wip/Tradershub/personalDetailsPopUpValidation.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    let country = Cypress.env("countries").CO
+  
+    beforeEach(() => {
+      localStorage.setItem("config.server_url", Cypress.env("stdConfigServer"))
+      localStorage.setItem("config.app_id", Cypress.env("stdConfigAppId"))
+      cy.c_visitResponsive("/endpoint", "desktop")
+      cy.findByRole("button", { name: "Sign up" }).should("not.be.disabled")
+      cy.c_enterValidEmail(sign_up_mail)
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wip/Tradershub/personalDetailsPopUpValidation.js:14]: I think this code is duplicate and used in many files, can we create a command for this?

**Review by bilalmajid-deriv — CHANGES_REQUESTED** [review]

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Review by bilalmajid-deriv — APPROVED** [review]
LGTM

**Review by bilalmajid-deriv — APPROVED** [review]

**Pull request merged**